presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/configs/ConfigHive3.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.product.launcher.env.configs;
+
+import com.google.common.collect.ImmutableList;
+import io.prestosql.tests.product.launcher.docker.DockerFiles;
+import io.prestosql.tests.product.launcher.env.Environment;
+
+import javax.inject.Inject;
+
+import java.util.List;
+
+import static io.prestosql.tests.product.launcher.env.EnvironmentContainers.PRESTO;
+import static java.util.Objects.requireNonNull;
+import static org.testcontainers.utility.MountableFile.forHostPath;
+
+public class ConfigHive3
```

Comment:
Is the environment capable of being used in the matrix here?
https://github.com/prestosql/presto/blob/dc1e5efdc1696a1d18957b62e6771d7c70bad171/.github/workflows/ci.yml#L283-L285

Reply:
I couldn't test the ci workflow locally, so image must be push first to test it remotely